### DISCUSSION TITLE
Time-Series for Classification and Time-Series for Regression

### DISCUSSION THREAD

**Discussion by drmingle** [winedarksea/AutoTS#53] [Ideas]
AutoTS currently provides forecasting, will we provide time series classification and time series regression, like other packages ([sktime](https://www.sktime.org/en/latest/))?

**Comment by winedarksea** [comment]
Forecasting and regression are basically two names for the same thing. Regression could theoretically include forecasting backwards in time - to handle that here, just inverse the time series. Some data manipulation and modification of your forecast_length and you can regress for almost any time you want.

However time series _classification_ is basically an entirely different field. There are some elements of classification here, but not many. Time series classification is often actually **speech recognition**. Speech being a time series of audio signals that needs to be classified to human words.
Forecasting and classification are so different that I don't see them being integrated here soon (albeit vaguely possible in the long term) - most packages out there specialize in either one or the other, and not both.

If you want to _forecast_ a class ie categorical data, that can already be done and is talked about in the extended_tutorial. To some people this would count as "classification" but I generally wouldn't call it that.
@drmingle
